fvm/evm/types/handler.go

```diff
@@ -15,17 +15,17 @@ import (
 // First, passing a signed transaction (EOA account) to the `EVM.run` Cadence function
 // creates a new block, updates the internal merkle tree, and emits a new root hash.
 //
-// The Second way is through a new form of account called bridged accounts,
+// The Second way is through a new form of account called cadence-owned-accounts (COAs),
```

Comment:
I did a rename on some of these but left the stdlib and fvm/evm packages for another PR since it was changing so many lines.